acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueProposeRpcAcceptanceTest.java

```diff
@@ -35,10 +35,10 @@ public void shouldAddValidators() throws IOException {
     cluster.start(minerNode1, minerNode2, minerNode3);
 
     cluster.verify(clique.validatorsEqual(minerNode1, minerNode2));
-    final WaitCondition cliqueValidatorsChanged = wait.cliqueValidatorsChanged(minerNode1);
+    final Condition cliqueValidatorsChanged = clique.awaitSignerSetChange(minerNode1);
     minerNode1.execute(cliqueTransactions.createAddProposal(minerNode3));
     minerNode2.execute(cliqueTransactions.createAddProposal(minerNode3));
-    cluster.waitUntil(cliqueValidatorsChanged);
+    cluster.verify(cliqueValidatorsChanged);
```

Comment:
This line should be removed - if we're ditching the "waiting" concept, then no need to ensure miner1 has changed - just ensure all the cluster have in the next line (and thus line 38 can probably go too).

Reply:
done